lib/MetaCPAN/Server/Controller/User.pm

```diff
@@ -5,6 +5,7 @@ use warnings;
 
 use DateTime;
 use Moose;
+use Log::Log4perl::MDC;
```

Comment:
? - why is this needed here?

Reply:
Later in the file it is used to attach some extra request data to the logging.